### DISCUSSION TITLE
I raise a case to GitHub support but no one respond to it. it's been more than 1 week

### DISCUSSION THREAD

**Discussion by mahsingSF** [community/community#147363] [New to GitHub]
I raise a case to GitHub support but no one respond to it. it's been more than 1 week

**Comment by Logophoman** [comment]
Did you post it here or write to the actual support? The questions here are answered by volunteers of the community...

**Reply by mahsingSF** [reply]
Thanks for replying my message.  i actually write to the actual support using this link - https://support.github.com/contact

**Reply by tuves** [reply]
Hi @mahsingSF,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
